ambry-store/src/test/java/com.github.ambry.store/IndexSegmentTest.java

```diff
@@ -793,9 +796,6 @@ private void getEntriesSinceTest(NavigableMap<MockId, NavigableSet<IndexValue>>
     highestIdIncluded = referenceIndex.lastKey().equals(idToCheck) ? null : referenceIndex.lastKey();
     // check the case where maxSize is more than the number of entries in the segment
     doGetEntriesSinceTest(referenceIndex, segment, idToCheck, maxSize + 1, 0, highestIdIncluded);
```

Comment:
One thing we can improve in the future is `doGetEntriesSinceTest` is invoked multiple times and even in the loop. I find that `doGetEntriesSinceTest` test both `getEntriesSince()` and `getIndexEntriesSince()` methods. Note that  `getIndexEntriesSince()` with `oneEntryPerKey = true` is actually tested in `getIndexEntriesSince()`. It seems we don't have to explicitly test `oneEntryPerKey = true` for `getIndexEntriesSince()` again. (My two cents.)

Reply:
For now I will leave the test since whether or not it is called is dependent on the implementation of `getEntriesSince` and `getIndexEntriesSince` is still exposed to the package. I'll think about how to further reduce the test length in future PRs.